.github/workflows/build-matrix.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
+        "OS": "ubuntu",
+        "NAMED_OS": "linux",
+        "RUNNER": "ubuntu-latest",
+        "ARCH": "x64",
+        "TARGET": "x86_64-unknown-linux-gnu"
+    },
+    {
+        "OS": "ubuntu",
+        "NAMED_OS": "linux",
+        "RUNNER": ["self-hosted", "Linux", "ARM64"],
+        "ARCH": "arm64",
+        "TARGET": "aarch64-unknown-linux-gnu",
+        "CONTAINER": "2_28"
```

Comment:
can be removed?

Reply:
We use it for pypi, no?

Reply:
yes, I haven't saw that you use this file for pypi as well when I commented it